### PR TITLE
Update terraform base project

### PR DESCRIPTION
**Pull request by pasqualedevita**
### Type of changes

- [ ] Add new resources
- [ ] Update configuration to existing resources
- [ ] Remove existing resources

### Does this introduce a change to production resources with possible user impact?

- [ ] Yes, users may be impacted applying this change
- [ ] No

### Does this introduce an unwanted change on infrastructure? Check terraform plan execution result

- [ ] Yes
- [ ] No

### Other information

---

### CODE REVIEW

**Review by ElfoLiNk — DISMISSED** [review]
pre commit hook also can generate docs automatically instead of manually executing terraform-docs

https://github.com/antonbabenko/pre-commit-terraform#notes-about-terraform_docs-hooks

File: README.md [hunk 1]
```diff
@@ -31,11 +31,11 @@ As PagoPA we build our standard Terraform modules, check available modules:
 To apply changes follow the standard terraform lifecycle once the code in this repository has been changed:
 
 ```sh
-terraform init
+terraform.sh init [uat|prod]
```
Comment (ElfoLiNk) [inline README.md]:
```suggestion
terraform.sh init [dev|uat|prod]
```

File: README.md [hunk 2]
```diff
@@ -31,11 +31,11 @@ As PagoPA we build our standard Terraform modules, check available modules:
...
 
-terraform plan
+terraform.sh plan [uat|prod]
```
Comment (ElfoLiNk) [inline README.md]:
```suggestion
terraform.sh plan [dev|uat|prod]
```

File: README.md [hunk 3]
```diff
@@ -31,11 +31,11 @@ As PagoPA we build our standard Terraform modules, check available modules:
...
 
-terraform apply
+terraform.sh apply [uat|prod]
```
Comment (ElfoLiNk) [inline README.md]:
```suggestion
terraform.sh apply [dev|uat|prod]
```

File: src/terraform.sh [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+if [ -z "$env" ]; then
+  echo "env should be: dev, uat or prod."
+  exit 0
+fi
+
+source "./env/$env/backend.ini"
+az account set -s "${subscription}"
+
+if echo "init plan apply refresh" | grep -w $action > /dev/null; then
```
Comment (ElfoLiNk) [inline src/terraform.sh]:
```suggestion
if echo "init plan apply refresh import" | grep -w $action > /dev/null; then
```

**Comment by pasqualedevita** [comment]
new updates in https://github.com/pagopa/terraform-infrastructure-template/pull/2

**Pull request closed without merging**